extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -137,6 +137,15 @@ The `amp-sidebar` component can be styled with standard CSS.
 -  The `width` of the `amp-sidebar` may be set to adjust the width between the pre-set min(45px) and max(80vw) values.
 - The height of the `amp-sidebar` may be set to adjust the height of the sidebar, if required. If the height exceeds 100vw, the sidebar will have a vertical scrollbar. The preset height of the sidebar is 100vw and can be overridden in CSS to make it shorter.
 - The current state of the sidebar is exposed via the `open` attribute that is set on the `amp-sidebar` tag when the side bar is open on the page.
+- The 'opacity' and 'background-color' of 'amp-sidebar-mask' may be set to adjust the opacity and background color of the mask. Include 'classList.add('amp-sidebar-mask')' to override the default style of 'i-amphtml-sidebar-mask'.
+
+Example:
+
```

Comment:
the example should be a CSS example. Remember this is JS inside our code, users have no idea about what this is.